specification/provisioningservices/resource-manager/Microsoft.Devices/2017-08-21-preview/iotdps.json

```diff
@@ -0,0 +1,1600 @@
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/provisioningServices/{provisioningServiceName}": {
+			"get": {
+				"tags": ["GET"],
+				"operationId": "IotDpsResource_Get",
+				"summary": "Get the non-security related metadata of a provisioning service.",
+				"description": "Get the non-security related metadata of a provisioning service.",
+				"parameters": [{
+					"name": "provisioningServiceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/subscriptionId"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ProvisioningServiceDescription"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			},
+			"put": {
+				"tags": ["PUT"],
+				"operationId": "IotDpsResource_CreateOrUpdate",
+				"summary": "Create or update the metadata of a provisioning service.",
+				"description": "Create or update the metadata of a provisioning service. The usual pattern to modify a property is to retrieve the provisioning service metadata and security metadata, and then combine them with the modified values in a new body to update the provisioning service.",
+				"x-ms-long-running-operation": true,
+				"parameters": [{
+					"$ref": "#/parameters/subscriptionId"
+				}, {
+					"name": "resourceGroupName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "provisioningServiceName",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}, {
+					"name": "ProvisioningServiceDescription",
+					"in": "body",
+					"required": true,
+					"schema": {
+						"$ref": "#/definitions/ProvisioningServiceDescription"
+					}
+				}, {
+					"$ref": "#/parameters/api-version"
+				}],
+				"responses": {
+					"200": {
+						"description": "This is returned as a response to the status polling request for the create or update operation. The body contains the resource representation that indicates a transitional provisioning state.",
+						"schema": {
+							"$ref": "#/definitions/Object"
+						}
+					},
+					"201": {
+						"description": "Created",
+						"schema": {
+							"$ref": "#/definitions/ProvisioningServiceDescription"
+						}
+					},
+					"default": {
+						"description": "DefaultErrorResponse",
+						"schema": {
+							"$ref": "#/definitions/ErrorDetails"
+						}
+					}
+				},
+				"deprecated": false
+			},
+			"delete": {
```

Comment:
operation not marked as long running. Please use x-ms-azure-long-running extension